module/clock/src/mod_clock.c

```diff
@@ -716,9 +732,10 @@ static int clock_process_event(const struct fwk_event *event,
         return clock_management_process_rate(event);
 #endif
 
+#ifdef BUILD_HAS_NOTIFICATION
```

Comment:
This is also not related to notifications and should be removed

Reply:
I rerun some test without notification and this one is not needed anymore. This patch is old and some changes happen in clock module since